frontend/src/metabase/querying/filters/components/FilterPicker/FilterColumnPicker/FilterColumnPicker.tsx

```diff
@@ -114,7 +124,7 @@ export function FilterColumnPicker({
         renderItemWrapper={renderItemWrapper}
         renderItemName={renderItemName}
         renderItemDescription={omitItemDescription}
-        renderItemIcon={renderItemIcon}
+        renderItemIcon={withColumnItemIcon ? renderItemIcon : undefined}
```

Comment:
I think we should make the usage of `withColumn...` props consistent - i mean here is that we use `withColumnGroupIcon` within the function in its own ternary, while here we3 declare the function undefined. can we move the `withColumnItemIcon` check into the `renderItemIcon` function?

Reply:
moved in https://github.com/metabase/metabase/pull/49768/commits/69abc1e99c4cf4c916389861c4d3f95f0e38568c, thanks!